README.md

```diff
@@ -14,7 +21,162 @@ We call this the fuel mix, and this fuel mix can impact on the carbon intensity
 
 Because the fuel mix will be different depending when and where you run your code, you can influence the carbon intensity of the code you write by moving it through time and space - either by making it run when the grid is greener, or making it run where it's greener, like a CDN running on green power.
 
-## Data
+## Installing
+
+- Install via [brew](https://brew.sh/).
+
+```sh
+brew install thegreenwebfoundation/carbon-aware-tools/grid-intensity
+```
+
+- Install via curl (feel free to do due diligence and check the [script](https://github.com/thegreenwebfoundation/grid-intensity-go/blob/main/install.sh) first).
```

Comment:
"Due diligence" is the best metaphor my poor 🧠 can come up with right now. 😅  

Feel free to suggest a snappier way to cover the trade offs with bash installers. 😄